modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc

```diff
@@ -0,0 +1,25 @@
+= Secure View
+
+Secure View is a mode where users can only read, and optionally comment on, files.
```

Comment:
- Commenting is not an option right now.
- You can optionally allow printing and exporting such files to PDF with watermarks included.